examples/aurora-mysql/local.tf

```diff
@@ -1,7 +1,15 @@
 locals {
-  cluster_name     = "sample-cluster-aurora-mysql"
-  environment      = "test"
-  cidr_block       = "172.16.0.0/16"
-  database_subnets = cidrsubnets(local.cidr_block, 8, 8, 8)
-  azs              = flatten(data.aws_availability_zones.available.names)
+  cluster_name              = "sample-cluster-aurora-mysql"
+  supporting_resources_name = "terraform-aws-rds-aurora"
+  tags = {
+    Name               = local.cluster_name
+    Environment        = "example"
+    "user::CostCenter" = "terraform-registry"
+    department         = "operations"
+    InstanceScheduler  = true
+    Project            = "aws-rds"
+    Owner              = "hugo.almeida"
+    LayerName          = "c550-aws-rds-cluster"
+    LayerId            = "c550"
+  }
```

Comment:
Should this follow the same latest formats of `cExample` and Owner as `Boldlink`